### ISSUE TITLE
TR/WCAG22/relative-luminance.html has no page `lang`

### ISSUE THREAD

**Issue by dd8** [w3c/wcag#3460]
https://www.w3.org/TR/WCAG22/relative-luminance.html has no `lang` attribute so fails WCAG 3.1.1 (level A)

It's linked to from Note 6 in:
https://www.w3.org/TR/WCAG22/#dfn-relative-luminance

It's also triggering an error in the W3 Validator due to a `<?xml` processing instruction on line 1. The page is also XHTML but is being served as text/html which contradicts the advice here:
https://www.w3.org/International/articles/serving-xhtml/

Probably better if the page was just plain HTML (i.e. remove the `<?xml` processing instruction and `xmlns=`) ...

**Edit:** ... unless the xml namespaces are required for backwards compatibility with pre-HTML5 MathML user agents.

**Comment by fstrr** [comment]
Good catch. I've created a PR for that which fixes it (and removes the xml content you mentioned)
